deploy/README.md

```diff
@@ -351,8 +351,49 @@ helm repo update \
    && helm install stable/fluent-bit --name fluent-bit --namespace sumologic -f fluent-bit-overrides.yaml
 ```
 
+## Step 4: Deploy Falco
```

Comment:
Should we say `Step 4: (Optional) Deploy Falco`

Reply:
Maybe, but we have dashboards dependent on this in the core Kubernetes app.

Reply:
Since Falco is not required for collection, but is required for the Kubernetes app, can we add something that communicates this in the doc? Maybe something like

`"This step is required for users who wish to use the Sumo Logic Kubernetes App."`

on line 356 would help make this clear.

Reply:
Makes sense, I was also thinking the same.

Reply:
How about this: "This step is required only if you intend to use the Sumo Logic Kubernetes App."

Reply:
I'm fine with that.

Reply:
Thanks, changed.